code/modules/assembly/signaler.dm

```diff
@@ -98,9 +98,9 @@
 			INVOKE_ASYNC(src, .proc/signal)
 			. = TRUE
 		if("freq")
-			frequency = unformat_frequency(params["freq"])
-			frequency = sanitize_frequency(frequency, TRUE)
-			set_frequency(frequency)
+			new_frequency = unformat_frequency(params["freq"])
+			new_frequency = sanitize_frequency(new_frequency, TRUE)
+			set_frequency(new_frequency)
```

Comment:
```suggestion
			var/new_frequency = sanitize_frequency(unformat_frequency(params["freq"]), TRUE)
			set_frequency(new_frequency)
```
Should just be just local variable instead of adding new parameter.

Reply:
Also from quick search, there's also a case of this with pressure plates, can you fix this as well ?

Reply:
AAnd it's also copypasted in signaler program.

Reply:
Right, yeah. I'll see if I can patch those.